packages/engine/Source/Renderer/CubeMap.js

```diff
@@ -14,6 +14,53 @@ import TextureMagnificationFilter from "./TextureMagnificationFilter.js";
 import TextureMinificationFilter from "./TextureMinificationFilter.js";
 
 /**
+ * @typedef CubeMap.BufferSource
+ *
+ * Exactly one of <code>arrayBufferView</code> or <code>bufferView</code> must be defined.
+ * @property {TypedArray} [arrayBufferView] A view of a binary data buffer containing pixel values.
+ * @property {TypedArray} [bufferView] A view of a binary data buffer containing pixel values, e.g. from a KTX2 file.
```

Comment:
There isn't a difference in the value, it's only a naming difference. The usual inputs use the name `arrayBufferView`. This is consistent with `Texture`. But KTX functions name the same property `bufferView`. This naming inconsistency is tangled into many of the glTF loader classes too.

Reply:
I updated the comment to try to explain the reasoning. Let me know if it is clearer now.

Reply:
Thanks for the explanation! It is clearer to me now, but I don't see why this function needs to support both property names. Why not enforce the one?

Reply:
For clarity, I didn't decide to support either--I'm just documenting the pre-existing behavior.

I think `CubeMap` was intended to be as similar to `Texture` as possible. `Texture` uses `arrayBufferView` everywhere.
But `CubeMap` frequently uses the raw output of the KTX parser, which names the field `bufferView`, consistently with other glTF-related stuff. So `CubeMap` accepts `bufferView` as well.

If we want to require one name, then there are 2 options:
1. Always use `bufferView` for `CubeMap` sources. This seems confusing for people working on the code later, especially if they have already worked on `Texture`.
2. Always use `arrayBufferView`. This would require translators for KTX-sourced data. A translation, in this case, involves looping through an array to copy 6 objects.

Reply:
Thanks. I didn't mean to imply a poor decision on your part here– Just that since this is being refactored and documented, we should tidy up where we can.

Aiming for consistency with `Texture` sounds best to me. In that case let's go with 2 if I understand correctly.

Reply:
As discussed offline, `KTX2Transcoder` returns an object composed of a `CompressedTextureBuffer` for each face in each mip level.

We added an alias `CompressedTextureBuffer.prototype.arrayBufferView` which points to the same private member accessed by the `bufferView` getter. Now the `CubeMap.BufferSource` type only needs an `arrayBufferView` property, and everything is more consistent between `CubeMap` and `Texture`. And we avoided the cost of copying objects to translate the key names.